lib/CLI/Types.hs

```diff
@@ -0,0 +1,134 @@
+-- SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+--
+-- SPDX-License-Identifier: MPL-2.0
+
+module CLI.Types where
+
+import Data.Text (Text)
+import Data.Time.Compat (Day, UTCTime , Year)
+import Data.Time.Calendar.Month.Compat (Month)
+import Entry (FieldKey, FieldVisibility, EntryTag)
+import Coffer.Path (Path, EntryPath)
+import Data.Set (Set)
+
+data Command
```

Comment:
Weren't they GADTs? Or is it the state before turning into GATDs?

Reply:
> Or is it the state before turning into GATDs?

Yup, they were turned into GADTs in the next PR.